locales/en.json

```
@@ -6,7 +6,7 @@
     "admin_password_changed": "The administration password got changed",
     "admin_password_too_long": "Please choose a password shorter than 127 characters",
     "already_up_to_date": "Nothing to do. Everything is already up-to-date.",
-    "app_action_cannot_be_ran_because_required_services_down": "This app requires some services which are currently down. Before continuing, you should try to restart the following services (and possibly investigate why they are down): {services}",
+    "app_action_cannot_be_ran_because_required_services_down": "Start all required services to run this app. Try to restart the following ones (and possibly investigate why they are down): {services}",
```

Hmmm to me the first part still looks weird ... It almost sounds like it's expected that these services are down, or that it was the responsability of  the admin to start these. But this message really is displayed in an *unexpected* situation where some required services are *unexpectedly* down, which probably is symptomatic of a larger issue about this service's configuration (e.g. an app messed up the syntax of the config file somehow).

Also stricto-sensu it's not about "running an app" and more "performing an action" (like install, remove, upgrade, backup, ...)

```suggestion
    "app_action_cannot_be_ran_because_required_services_down": "It appears that some services required to perform this action are down. Try to restart the following ones (or investigate why they are down): {services}",
```